packages/itwin/imodel-react-hooks/package.json

```diff
@@ -50,14 +54,15 @@
     "verbose": true
   },
   "devDependencies": {
-    "@bentley/bentleyjs-core": "^2.0.0",
-    "@bentley/geometry-core": "^2.0.0",
-    "@bentley/imodeljs-common": "^2.0.0",
-    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@itwin/build-tools": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
     "@testing-library/react": "^9.1.4",
     "@types/jest": "^24.9.0",
-    "@types/react": "~16.9.35",
-    "@types/react-dom": "^16.8.0",
+    "@types/react": "^17.0.19",
```

Comment:
should this be uniform with core's ui packages which are using "^17.0.0" ?

Reply:
its just types so should be ok; theyll probably resolve to the same dep because of the ^